invoke-ai/InvokeAI
# Bugfix/ignore dot directories on model scan

## What type of PR is this? (check all applicable)



## Have you discussed this change with the InvokeAI team?

      
## Have you updated all relevant documentation?


## Description

Mac users have a recurring issue in which a `.DS_Store` directory is created in their `models` hierarchy, causing the new model scanner to freak out. This PR skips over any paths that begin with a dot. I haven't tested it on a Macintosh, so I'm not 100% certain it will do the trick.

## Related Tickets & Documents

- Related Issue #4815 

## QA Instructions, Screenshots, Recordings

Someone with a Mac please try to reproduce the `.DS_Store` crash and then see if applying this PR addresses the issue.
